Anton.Konovalov/Homework8/module.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'pry'
```

Comment:
you don't need this gem anymore, this was needed only for development, pls, remove it

Reply:
+